packages/veritone-widgets/src/widgets/SourceManagementForm/index.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { arrayOf, object, objectOf, any, func } from 'prop-types';
+import { SourceManagementForm } from 'veritone-react-common';
+
+import widget from '../../shared/widget';
+
+class SourceManagementFormWidget extends React.Component {
+  static propTypes = {
+    sourceTypes: arrayOf(object).isRequired,
+    templateData: objectOf(any).isRequired,
+    source: objectOf(any),
+    initialTemplates: objectOf(any),
+    onSubmit: func.isRequired,
+    onClose: func
```

Comment:
This is a required prop in `SourceManagementForm` so should also be required here

Reply:
fixed shouldn't be required